.github/workflows/integration.yml

```diff
@@ -25,14 +25,13 @@ jobs:
         # just ubuntu and windows give enough confidence
         # osx should work fine (and we test that locally often)
         os: [ubuntu, windows]
-        # just 1 version, it's heavy
-        python-version: [3.8]
-        ipywidgets: ["7.7", "8.0"]
         include:
-          - ipywidgets: "7.7"
-            voila: "0.3.0"
-          - ipywidgets: "8.0"
-            voila: "0.4.0"
+          - ipywidgets: "7.8"
+            voila: "0.3.7"
+            python-version: "3.10"
+          - ipywidgets: "8.1"
+            voila: "0.5.4"
+            python-version: "3.10"
```

Comment:
Note that we use a ~ for the installation, so this change is not needed.

Reply:
Ah, I see that you have changed the matrix from 2x2 to 1x2. I think ipywidsgets should be kept top level for the matrix to work.